build/python-client/Dockerfile

```diff
@@ -4,6 +4,11 @@ FROM --platform=x86_64 python:${PYTHON_VERSION}-buster
 
 RUN mkdir /proto
 
-RUN pip install grpcio grpcio-tools mypy-protobuf
+# Project Install
+COPY client/python/pyproject.toml /code/pyproject.toml
+RUN pip install /code\[test\] --use-feature=in-tree-build
```

Comment:
I don't really understand this.

Reply:
This is what I had to do to fix the code last night in CI 😅. Would you want comments written about specifically why each thing is done or is discussing it in a meeting better?

Reply:
Comments will live forever! :)

Reply:
On it :))